include/itkTileMontage.h

```diff
@@ -119,6 +119,18 @@ class ITK_TEMPLATE_EXPORT TileMontage : public ProcessObject
   itkSetMacro( ForcedSpacing, SpacingType );
   itkGetConstMacro( ForcedSpacing, SpacingType );
 
+  /** Set/Get absolute registration threshold.
+   * The maximum allowed residual error for a registration pair during
+   * global optimization. Expressed in number of pixels. Default: 1.0. */
```

Comment:
Please make a note here about what happens when a registration pair's error exceeds the threshold.